specification/network/resource-manager/Microsoft.Network/preview/2021-02-01-preview/examples/NetworkManagerSecurityConfigurationEvaluateImport.json

```diff
@@ -24,92 +24,82 @@
       "body": {
         "value": [
           {
-            "securityType": "UserPolicy",
-            "securityUserRule": {
-              "id": "/subscriptions/subId/resourceGroups/rg1/providers/Microsoft.Network/networkmanagers/testNetworkManager/securityConfigurations/myTestConfig/userRules/SampleRule",
-              "type": "Microsoft.Network/networkManagers/securityConfigurations/userRules",
-              "name": "SampleRule",
-              "systemData": {
-                "createdBy": "b69a9388-9488-4534-b470-7ec6d41beef5",
-                "createdByType": "User",
-                "createdAt": "2021-01-11T18:52:27Z",
-                "lastModifiedBy": "b69a9388-9488-4534-b470-7ec6d41beef5",
-                "lastModifiedByType": "User",
-                "lastModifiedAt": "2021-01-11T18:52:27Z"
-              },
-              "properties": {
-                "description": "Sample user rule",
-                "protocol": "Tcp",
-                "source": [
-                  {
-                    "addressPrefixType": "IPPrefix",
-                    "addressPrefix": "*"
-                  }
-                ],
-                "destination": [
-                  {
-                    "addressPrefixType": "IPPrefix",
-                    "addressPrefix": "*"
-                  }
-                ],
-                "sourcePortRanges": [
-                  "0-65535"
-                ],
-                "destinationPortRanges": [
-                  "22"
-                ],
-                "direction": "Inbound",
-                "provisioningState": "Succeeded"
-              }
+            "id": "/subscriptions/subId/resourceGroups/rg1/providers/Microsoft.Network/networkManagers/testNetworkManager/securityAdminConfigurations/adminConfig/rules/SampleRule",
```

Comment:
where is collection?

Reply:
should be adminConfig/collections/xxx/rules/SampleRule? because rule is child obj of colletion

Reply:
Same Q below

Reply:
Good catch, will add it